src/modules/display-movies.js

```diff
@@ -1,28 +1,31 @@
 import coming from '../assets/coming-soon.jpg';
-import getliked from './get-liked.js';
-import { addLike, getLikes } from './likes.js';
 import renderPopup from './render-popup.js';
+import handleLikeButtonClick from './likes.js';
 
-// Function to display movies based on the selected genre
-const displayMovies = (movieArray) => {
+const displayMovies = async (movieArray) => {
   const container = document.querySelector('main');
-  container.innerHTML = ''; // Clear the current movies displayed on the page
+  container.innerHTML = '';
 
   if (!movieArray || movieArray.length === 0) {
-    // Handle the case when genreData is not available or empty
     const noMoviesMessage = document.createElement('p');
     noMoviesMessage.textContent = 'No movies found for this category.';
     container.appendChild(noMoviesMessage);
     return;
   }
 
-  movieArray.forEach((movie) => {
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < movieArray.length; i++) {
```

Comment:
I see you are trying to avoid the Eslint error and you managed to avoid it ✔️ 

But there is also an easier way to do it:
simply use `i += 1` instead of `i++` and you will not need to add this comment. Eslint allows the `+=` syntax 💯 

This will help you in future by saving you time 🕐  fixing Eslint errors 😃

Reply:
Thanks, I have been struggling it